### DISCUSSION TITLE
New Quarto project from version control? (Rstudio)

### DISCUSSION THREAD

**Discussion by sierra-raby** [quarto-dev/quarto-cli#4847] [Q&A] [answered]
I am trying out Quarto and can't get past project setup from version control. I am trying to use the same approach as creating a new R project in R studio with an existing github repo. For a new R project, you simply follow New Project -> Version Control -> Git from within R studio and connect to your existing (newly created in my case) github repo. How can I do this for a new Quarto project?

**Comment by eteitelbaum** [comment, marked as answer]
I had the same question. It is a little confusing. There are three options for creating a new project: New Directory; Existing Directory; and Version Control. If you select New Directory you see options for Quarto Project, Quarto Blog, Quarto Website, etc. You do not see these options if you select Existing Directory or Version Control as methods for creating a new project. 

You can still follow your old workflow with the Version Control option. You just need to create the Quarto documents in the new version control directory. So you would go New Project > Version Control > Git and create the project. From there, click on File > New File > Quarto Document and then choose Document, Presentation or Interactive based on what kind of document you want to create.  You can do the same thing for websites, blogs and books but then you need to manually insert the various additional component files like the _quarto.yaml, styles.css and index.qmd (for a website for example). 

The other option is to create the project from New Directory and select the kind of project you want to create (document, website, blog or book). Before you hit Create Project be sure select "Create git repository." Then once the project is open you can type `usethis::use_github()` to generate a repo on GitHub for the project. Full details of this workflow are provided in chapter 17 of [Happy Git and GitHub for the R UseR](https://happygitwithr.com/existing-github-last.html).
